### DISCUSSION TITLE
Add a field

### DISCUSSION THREAD

**Discussion by danielgap** [omkarcloud/google-maps-scraper#55] [Q&A]
I want to add a new field to the results, but I can't do it. For example if I want to add a field to the results that is called email, although for the example it is only in all email@email.com, where would I add it.

**Comment by Chetan11-dev** [comment]
You need to modify the `clean` function within the `scrape_google_maps_links_task.py` file to incorporate a new field. For reference, please see the screenshot: ![Screenshot (207)](https://github.com/omkarcloud/google-maps-scraper/assets/53407137/4a45b8ed-7ba2-43d8-ac65-290bc818ab95).
